README.md

```diff
@@ -2,7 +2,7 @@
 
 ## :information_source: Overview
 - :world_map: An add-on plugin for *Towny*, which makes claiming more organized, reducing staff workload and server toxicity.
-  
```

Comment:
🚧 
- Normally I don't even mention accidental extra spaces but in this specific case .... an extra space can (weirdly) make big difference to how to the help page displays. Please revert.

Reply:
* If you can fix this, great, if not I can quickly patch it after merging.